### ISSUE TITLE
[Feature Request] Adjustable left sidebar (all/downloading/finished/seeding/stopped/failed/labels)

### ISSUE THREAD

**Issue by X73RN4L** [tympanix/Electorrent#350]
Possibly implement:

- the ability to resize the left sidebar that contains: "All, Downloading, Finished, Seeding, Stopped, Failed & Labels"

(adjusting the settings 'display size' & 'table resize style' does not resize the left sidebar.)

**Comment by Asinin3** [comment]
Yeah.. Really need this, also need a way to hide it.. its so huge and i dont need to see it.

**Comment by tympanix** [comment]
Partially addressed in v2.10.0 by #521, which makes the sidebar collapsible. Arbitrary sidebar-width resizing is not implemented, so this remains open.

**Comment by tympanix** [comment]
Sidebar size reduced (padding/margin) to make more compact.

New collapse feature implemented.

Custom width resizing considered out of scope.
